### DISCUSSION TITLE
Background

### DISCUSSION THREAD

**Discussion by fanynek86** [Ylianst/MeshCentral#6509] [Q&A]
Hello gentlemen!
Any ideas on how to turn off the background on the splash screen so it's just black? Alternatively, how to set the black frame after logging in. I'd say there's a native blue background saved everywhere.

Thanks for any idea.

**Comment by si458** [comment]
You need to really show pictures as to what u mean? What splash screen? What background? What black frame?

**Comment by fanynek86** [comment]
In the marked places, change the background, whether it is an image or a color.
![MC_background](https://github.com/user-attachments/assets/c2b81990-50ee-4f41-9f3d-bbea16c9e66f)

**Reply by si458** [reply]
unfortunately they are hard coded,
you would need to edit the `login2.handlebars` and `default.handlebars` files
and then every update of meshcentral, edit them again,
unless u can edit the `style.css` and add the colours in there in with the likes of `!important`

**Comment by fanynek86** [comment]
Thank you very much anyway :-)! I appreciate your helpfulness and quick responses.
